test_regress/t/t_enum_int.pl

```diff
@@ -11,6 +11,7 @@
 scenarios(simulator => 1);
 
 compile(
+    v_flags => ['--x-initial-edge']
```

Comment:
Same as above, shouldn't need this.

Reply:
Seems to be a leftover from earlier in development. I'll remove it.